core-bundle/src/EventListener/DbafsMetadataSubscriber.php

```diff
@@ -98,4 +112,25 @@ private function supports(AbstractDbafsMetadataEvent $event): bool
     {
         return 'tl_files' === $event->getTable();
     }
+
+    private function getDefaultLocales(): array
+    {
+        if (null !== $this->defaultLocales) {
+            return $this->defaultLocales;
+        }
+
+        $page = $this->requestStack->getMainRequest()?->attributes?->get('pageModel');
```

Comment:
```suggestion
        $page = $this->requestStack->getMainRequest()?->attributes->get('pageModel');
```

If there is a request object, it will have an `attributes` property.

Reply:
True ☺️